server/test/common/data/test.raml

```diff
@@ -0,0 +1,47 @@
+#%RAML 1.0
+title: API with Examples
+
```

Comment:
What's a RAML file doing here? in this PR?

Reply:
Added this to test invalid extension handling.